lib/src/intl_suggestors/utils.dart

```diff
@@ -0,0 +1,346 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/type.dart';
+import 'package:file/file.dart';
+import 'package:over_react_codemod/src/util/component_usage.dart';
+import 'package:collection/collection.dart' show IterableExtension;
+
+import 'constants.dart';
+
+// -- Node Validation --
+bool isValidStringInterpolationNode(AstNode node) {
+  if (!(node is StringInterpolation)) return false;
```

Comment:
FYI you can use `is!` for these checks
```suggestion
  if (node is! StringInterpolation) return false;
```

Reply:
![](https://media.giphy.com/media/WfBZwNA6XSjphkYkzN/giphy.gif)